Cantoboard/Extension/URL+Extension.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  URL+Extension.swift
+//  Cantoboard
+//
+//  Created by Alex Man on 9/12/21.
+//
+
+import UIKit
+import AVFoundation
+
+extension URL {
+    var videoAspectRatio: CGFloat? {
```

Comment:
This operation looks super expensive. Could we cache them instead?

Reply:
Since all the video are in the same size currently, hard-code it until then.